src/main/java/org/scijava/log/IgnoreAsCallingClass.java

```diff
@@ -0,0 +1,10 @@
+package org.scijava.log;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * @author Matthias Arzt
+ */
+@Retention(RetentionPolicy.RUNTIME)
+public @interface IgnoreAsCallingClass {}
```

Comment:
Please improve the commit message to describe more explicitly what this commit is all about, though!